test/sql/error/test_try_expression.test

```diff
@@ -0,0 +1,115 @@
+# name: test/sql/error/test_try_expression.test
+# group: [error]
+
+statement ok
+pragma enable_verification;
+
+# --- OPERATOR_CAST ---
+
+query I
+select TRY('abc'::INTEGER)
+----
+NULL
+
+# --- OPERATOR_NOT ---
+
+## FIXME: this should error?
+#query I
+#select ~CAST('abc' as INTEGER)
```

Comment:
Is this not erroring in the unit test runner somehow? It seems to error in the CLI

Reply:
Seems to error now, not sure what was happening before, perhaps it was recently fixed by some other change?